.gitignore

```diff
@@ -7,6 +7,7 @@
 # Our build scripts - necessary for our release jobs
 hibernate-noorm-release-scripts
 hibernate-release-scripts
+jreleaser
```

Comment:
This is probably no longer needed, since the scripts now "install" jreleaser in `hibernate-release-scripts`?